chairman/configuration/defaults/simpleview/genesis/delegate-keys.005.key

```diff
@@ -0,0 +1 @@
+X�����C��%��Y����G�c�b~�L�n�<.���J�xH�Q��V���钮�C�Ӷ�Q�&j%�" 7��F7ҥ�a����6�[��/=cK41(i���@�W����f��w�����
```

Comment:
@newhoggy  The genesis files should be generated on the fly so these need to be removed.

Reply:
Yep, I'm working on the file generation at the moment.